Assets/Scripts/Notebook/Button.cs

```diff
@@ -71,6 +71,9 @@ private void OnEnable()
             _teleportController = TeleportController.Instance;
             if (_teleportController == null) throw new System.Exception("Must have a teleport controller in the scene");
 
+            _rightColorController = ColorController.Instance;
+            if (_rightColorController == null) throw new System.Exception("Must have a color controller in the scene");
+
```

Comment:
I think we should go back to dragging it in. I don't think a ColorController should be necessary